src/EditorFeatures/VisualBasic/Wrapping/WrappingIndentationService.vb

```diff
@@ -0,0 +1,22 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports Microsoft.CodeAnalysis.Editor.VisualBasic.Formatting.Indentation
+Imports Microsoft.CodeAnalysis.Formatting.Rules
+
+Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.Wrapping
+    Friend Class WrappingIndentationService
+        Inherits VisualBasicIndentationService
```

Comment:
@sharwell Turns out this is why we can't use a singleton for the BlankLineIndentationService.   I actually depend on being able to override its behavior to get the wrapping behavior we want.

Reply:
sure.